### PR TITLE
Added the batchexecutor takes the runas user instead of the logged in

### PR DESCRIPTION
**Pull request by shazada**
user.

This is needed when some tasks should be done by System or in case you've used the batchexecutor in a webscript where a runas user is specified.

### CODE REVIEW

File: src/main/java/nl/ciber/alfresco/repo/jscript/batchexecuter/ScriptBatchExecuter.java [hunk 1]
```diff
@@ -123,7 +123,8 @@ public synchronized boolean cancelJob(String jobId) {
             runningJobs.put(job.getId(), job);
 
             final Scriptable cachedScope = getScope();
-            final String user = AuthenticationUtil.getFullyAuthenticatedUser();
+            //final String user = AuthenticationUtil.getFullyAuthenticatedUser();
```
Comment (byaminov) [inline src/main/java/nl/ciber/alfresco/repo/jscript/batchexecuter/ScriptBatchExecuter.java]: @shazada could you please remove this commented out line?

Comment (shazada) [inline src/main/java/nl/ciber/alfresco/repo/jscript/batchexecuter/ScriptBatchExecuter.java]: done!

**Comment by byaminov** [comment]
Oops, I don't seem to have access rights on this repo anymore. @jpzwarte or @rvermeire could you please give me push access on this repo? Or if you are not interested in it, then you could also just transfer it to my GitHub user :)

**Comment by jpzwarte** [comment]
@byaminov hey Bulat, sorry for the late response. I'm trying to transfer it to your personal account, but github doesn't allow that.

**Comment by shazada** [comment]
Hi @jpzwarte You can just add Bulat as a contributor I guess.

**Comment by jpzwarte** [comment]
@shazada done!

**Comment by byaminov** [comment]
@jpzwarte thanks, but I still don't have push access on this repo. Can you please check again?

**Comment by byaminov** [comment]
@jpzwarte sorry, my bad! Had to accept the invitation first!

**Comment by byaminov** [comment]
@shazada now we have an extra commit in this PR which is not supposed to be merged I guess: d8987171fdb519ecf34723fe402b270fb37e2066. Can you please remove it from your master, or create a new PR from a new branch in your fork where you don't have it? :)

It's indeed funny how much conversation we have to have for a one-line change! :)

**Comment by shazada** [comment]
Haha Oeps, this suppose to be internally :).

Yes I was testing something due to releasing, didn't work though...

**Comment by shazada** [comment]
Done!

**Pull request merged**